woocommerce/woocommerce-android
# TTP: Moves the reader type selection from the dialog to the payment method selection screen

Closes: #8434


### Description

The PR:
* Removes card reader type selection dialog
* Splits "card" payment method type into "Tap To Pay" and "Bluetooth reader"
* Changes color of the icons on the Payment Method selection screen. They had different colors 🤷‍♂️

### Testing instructions

* Run regression testing of IPP and TTP flow
* Notice that the card reader type selection dialog is gone and now BT or TTP can be chosen on the Payment Method selection screen
* Notice that when IPP is not available for an order then both options should be not visible on the Payment Method selection screen
* Notice that when IPP is available for the order but TTP is not, then BT reader is available while TTP not
* Notice that if the Built order is connected but BT flow is chosen then the Built-In reader gets disconnected and visa versa
* Run regression testing of Payment Hub -> Onboarding flows

### Images/gif



https://user-images.githubusercontent.com/4923871/222358276-c3a4e375-035a-4ed6-b88e-946bce4824ed.mp4
